src/app/components/home/TransactionPanel.tsx

```diff
@@ -413,7 +412,8 @@ const RowItem: React.FC<{ tx: Transaction }> = ({ tx }) => {
           {content}
         </Typography>
         <Link
-          href={getTrackerLink(networkId, hash, 'transaction')}
+          // TODO
+          href={getTrackerLink(0x7, hash, 'transaction')}
```

Comment:
This is hardcoded to Berlin

Reply:
@shallcdn Please help me answer this question. Thank you!